.bazelrc

```diff
@@ -27,37 +27,10 @@ build --config=short_logs
 # PyTorch/XLA uses exceptions to communicate with Python.
 build --copt=-fexceptions
 
-# Force GCC because clang/bazel has issues.
-build --action_env=CC=gcc
-build --action_env=CXX=g++
-build --spawn_strategy=standalone
+build --spawn_strategy=sandboxed
```

Comment:
I made the same change but had to revert it (https://github.com/pytorch/xla/pull/9212) as it slowed the local build drastically...